types/memmonit/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for memmonit 1.1
+// Project: https://github.com/diegodamato/memmonit
+// Definitions by: Rodrigo Ribeiro <https://github.com/ribeiro-rodrigo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
```

Comment:
These need to be inside namespace instead.. 
https://github.com/diegodamato/memmonit#usage-examples shows the usage as 
```
let memmonit = require("memmonit").MemoryMonitoring; // Note this

let memoryUsage = memmonit.getMemoryUsage("GB");
```